ajax/homepage_wizard.php

```diff
@@ -53,7 +53,7 @@
 }
 
 function plugin_formcreator_showWizardCategories() {
-   $tree = PluginFormcreatorCategory::getCategoryTree(0, false);
+   $tree = PluginFormcreatorCommon::getCategoryTree(0, false);
```

Comment:
```suggestion
   $tree = PluginFormcreatorCommon::getCategoryTree(0, false);
```
Maybe change also in `ajax/kb_category.php`